pr_insight/algo/utils.py

```
@@ -949,7 +953,7 @@ def validate_and_await_rate_limit(github_token):
                     time.sleep(sleep_time_sec + 1)
                 rate_limit_status = get_rate_limit_status(github_token)
         return rate_limit_status
-    except Exception:
+    except:
```

Ruff

Do not use bare `except`

(E722)